tiatoolbox/tools/stainextract.py

```diff
@@ -14,35 +14,207 @@
 # along with this program; if not, write to the Free Software Foundation,
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 #
+# This file contains code inspired by StainTools
+# [https://github.com/Peter554/StainTools] written by Peter Byfield.
+#
 # The Original Code is Copyright (C) 2020, TIALab, University of Warwick
 # All rights reserved.
 # ***** END GPL LICENSE BLOCK *****
 
-"""Stain normalisation utilities used in tiatoolbox"""
+"""Stain normalisation stain matrix extraction."""
```

Comment:
Stain matrix extraction for stain normalisation.

Reply:
Fixed